frontend/src/helpers/clusterSettings.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Cluster settings
+ * This file contains functions to store and load cluster settings in local storage.
+ */
+export interface ClusterSettings {
```

Comment:
This looks like a comment for the interface, maybe add different comment or just add a new line before it? WDYT?

Reply:
Fixed.